dspace/config/modules/authentication-password.cfg

```diff
@@ -6,6 +6,9 @@
 #---------------------------------------------------------------#
 #
 
+# self-registration can be disabled completely by setting the user.registration property to false
+# user.registration = false
```

Comment:
Forgot to add a comment here.  While it's good to add this setting in our configs, I noticed it doesn't seem to work.  But, I'm assuming this doesn't work because #3272 isn't fixed yet?

Reply:
No, this does work, I think you misread `user.registration = false` (commented out) mean you can't register, setting it to true (the default) would make it so registration doesn't work

- See https://github.com/DSpace/dspace-angular/pull/2016: With config set (in `local.cfg`): `user.registration=false` => `{ui.url}/register` not accessible (redirects login or forbidden if logged in) & button to it in login dropdown not present

Reply:
@MarieVerdonck : You were correct. I completely misread this config :)  tested again today and it works exactly as expected. When set to `user.registration=false`, I see registration is disabled/turned off, and I no longer see that option in the UI level.  So, this proves to me that this PR *fixes* #3272.